crates/services/p2p/src/service.rs

```diff
@@ -89,8 +89,6 @@ pub type Service<V> = ServiceRunner<UninitializedTask<V, SharedState>>;
 enum TaskRequest {
     // Broadcast requests to p2p network
     BroadcastTransaction(Arc<Transaction>),
-    // Request to get one-off data from p2p network
-    GetPeerIds(oneshot::Sender<Vec<PeerId>>),
```

Comment:
It is not used, so I removed it=)

Reply:
it would be cleaner to make this kind of change in a separate PR, as it's totally unrelated to backwards compatibility right?

Reply:
https://github.com/FuelLabs/fuel-core/pull/1913